indy-besu/docs/roadmap.md

```diff
@@ -0,0 +1,42 @@
+# Hyperledger Indy Besu ROADMAP
+
+Note: Right now we have finished PoC implementation. Roadmap tasks and their priorities are still in process of defining.
+
+## Phase 0: PoC
+* Basic transactions (without additional validation) 
+* VDR
+* Migration design 
+* Experiments (eth signature, upgrade, role model, migration)
+
+## Phase 1: MVP
+* Aries Framework Javascript integration Demo
+* Double-check did:indy, did:sov methods compatibility
+* Add `DidUniversalResolver` contract so that adding a support for new DID method
+* Implement a validation to check VerificationRelationship format (Can accept only id or verificationMethod)
+* Validate the DID identifier using its associated verification key (will probably need to be implemented sha2 hash and Base58 encoder)
```

Comment:
> implemented sha2 hash and Base58 encoder)

Is it am easy task? If not, I would move it to next phases.

Reply:
Moved